scripts/run_performance_job.py

```diff
@@ -300,7 +378,7 @@ def run_performance_job(args: RunPerformanceJobArgs):
         env_var_name = "PerfCommandUploadToken" if args.os_group == "windows" else "PerfCommandUploadTokenLinux"
         args.perflab_upload_token = os.environ.get(env_var_name)
         if args.perflab_upload_token is None:
-            print(f"WARNING: {env_var_name} is not set. Results will not be uploaded.")
+            getLogger().warn(f"{env_var_name} is not set. Results will not be uploaded.")
```

Comment:
Not sure how you want to treat this, but this will need to be changed when we disable the SAS token loading. Currently, we are not even using the SAS token, but it is still loaded into the environment.

Reply:
I've edited the message. Once we get rid of SAS token loading we can get rid of all references to perflab_upload_token from the codebase.